common/djangoapps/edeos/api_calls.py

```diff
@@ -140,9 +145,13 @@ def post(self, url, payload, headers=None, can_retry=True):
         if headers is not None:
             headers_.update(headers)
         resp = requests.post(url, json=payload, headers=headers_)
-        log.info("Edeos response: status {}, content {}".format(resp.status_code, resp.content))
+        log.info("Edeos response: status {}, content {}".format(resp.status_code, resp.content)) #TODO this response will be in the termilall and it is not convenient maybe not in the teminal? 
```

Comment:
Please explain why this log isn't convenient; not sure I understand...
Or maybe remove this TODO :)

Reply:
resolved